### PR TITLE
docs: refactor

### PR DESCRIPTION
**Pull request by arnupretorius**
## What?
Small PR to clean up the landing page. Move some of the markdowns to the docs folder and the notebook to an examples folder.

## Why?
Simpler and easier to read.

### CODE REVIEW

**Review by RuanJohn — CHANGES_REQUESTED** [review]
Thank you @arnupretorius :raised_hands: 
One minor detail, can you also please fix the link in the starter notebook itself for launching on Colab.

**Review by RuanJohn — DISMISSED** [review]
Thanks @arnupretorius :raised_hands:

File: README.md [hunk 1]
```diff
@@ -161,7 +158,7 @@ If you use Mava in your work, please cite the accompanying
 ```
 
 [Paper]: https://arxiv.org/pdf/2107.01460.pdf
-[quickstart]: https://github.com/instadeepai/Mava/blob/develop/Quickstart.ipynb
+[quickstart]: https://github.com/instadeepai/Mava/blob/develop//examples/Quickstart.ipynb
```
Comment (OmaymaMahjoub) [inline README.md:161]: Minor improvement (although it works without changing it):  `develop//examples/` -> `develop/examples/`

**Review by OmaymaMahjoub — DISMISSED** [review]
Looks good! Just minor change that we can skip if we think it is unnecessary :)

**Review by OmaymaMahjoub — APPROVED** [review]

**Review by RuanJohn — APPROVED** [review]

**Pull request merged**